tinyfont.go

```diff
@@ -82,8 +88,31 @@ func WriteLineRotated(display drivers.Displayer, font *Font, x int16, y int16, t
 	rotation = rotation % 4
 	w, h := display.Size()
 	l := len(text)
+	ox := x
+	oy := y
 	for i := 0; i < l; i++ {
-		glyph := font.Glyphs[byte(text[i])-font.First]
+		if text[i] == 0x0A || text[i] == 0x0D {
```

Comment:
Please use named constants instead of using _"magic numbers"_

Reply:
Fixed. (a6bb83a)